### DISCUSSION TITLE
Those alternatives might provide privacy but they don't provide any freedom

### DISCUSSION THREAD

**Discussion by ElectroidDC** [pluja/awesome-privacy#312] [Discussion]
I'm talking about service like F-Droid that in their statement said "free speech" is not a good idea and "F-Droid takes political stance"
https://f-droid.org/en/2019/07/16/statement.html

And Mastodon that disabled on app level ability to connect to Gab and calls that "a feature"
https://github.com/tuskyapp/Tusky/issues/1381

What is the point of service that offers "privacy", that is "open sources", that is "decentralized" but at the same time censorship is imposed by some authority? You surely have a lot of privacy when you have no access to service but I guess that's no the point here.

To add more examples and to show that both sides of political spectrum can be excluded from both of those services:
https://blog.alexgleason.me/fdroid-banned-spinster/

We might just use Google Play and Twitter and save ourselves the trouble.

**Comment by pluja** [comment]
Hi @RogerOne101 

In the first place, privacy and freedom are not interdependent. 

It is a fact that Mastodon respects the user's privacy much more than Twitter or Facebook. There are plenty of alternative clients that do not censor Gab at all, or you can use Pleroma if you prefer. That's the nice thing about free and/or open source software and decentralization. On the other hand, if Twitter censors (and it does), you have nothing to do. You are also free to start your own community server in Mastodon/Pleroma and apply your own rules. Nostr is also another great alternative.

If F-Droid's political stance does not align with your beliefs, there are other options like Obtainium or downloading apps directly from the source. However, it is important to note that every platform has the right to take a stance on issues that matter to them, and users are free to decide whether to support those platforms based on their beliefs. I myself do not use F-Droid and have switched to Obtainium as it provides me more control.

I understand that you are concerned about censorship, but it is essential to recognize that imposing censorship on platforms that do not align with our personal beliefs is not the solution. Likewise, I am completely against censorship, and for this reason, I think that censoring content just because it does not align with my viewpoint is wrong.

If you want to use Google Play and Twitter, you can do it. But it does not make much sense from the privacy point of view; by using these services, you are losing your privacy and freedom, as these platforms are highly censored, centralized, and apply plenty of algorithmic manipulation techniques.

**Comment by ElectroidDC** [comment]
Revisiting post after 3 years.

F-Droid’s recent statement condemning Google’s efforts to restrict sideloading and third-party app stores perfectly highlights their hypocrisy.

F-Droid proudly declared in 2019 that "free speech is not a good idea" and that they take a political stance, banning or excluding apps and developers they disliked. They positioned themselves as gatekeepers under the banner of "safety" or "ethics." Now that a bigger centralized power (Google) is turning the same exclusionary tactics against them, suddenly it's a threat to the ecosystem and they need community support.

F-Droid fought against broad platform freedom. They finally got what they fought for.
